CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Updated Substrate to polkadot-v0.9.27
+- DDC staking requirements now distinct for each DDC cluster
+- DDC participant stake chilling is delayed if the delay is not explicitly set to zero
```

Comment:
`4.5.0` has been already released, let's put it under `vNext`

Reply:
Thanks, updated.